testdata/storage/localvolume/sub.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: local-storage-operator
+spec:
+  channel: "4.6"
```

Comment:
@qinpingli perhap this should be a variable?  Or are you  patching it in your test?

Reply:
Yes, I updated it in the step with the following code:
```
# get channel
step %Q/evaluation of `cluster_version('version').channel.split('-')[1]` is stored in the :channel clipboard/
step %Q/I run oc create over "sub.yaml" replacing paths:/, table(%{
      | ["metadata"]["name"] | #{name}         |
      | ["spec"]["channel"]  | "#{cb.channel}" |
    })
```